.github/workflows/publish.yml

```diff
@@ -0,0 +1,49 @@
+name: Publish to PyPI
+
+# on:
+#   release:
+#     types:
+#       - published
+on:
+  pull_request:
+
+jobs:
+  packages:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+
+    - name: Set up Python
+      uses: actions/setup-python@v1
+      with:
+        python-version: 3.x
+
+    - name: Get tags
+      run: git fetch --depth=1 origin +refs/tags/*:refs/tags/*
+      shell: bash
+
+    - name: Install build tools
+      run: |
+        python -m pip install --upgrade pip wheel "setuptools>=41.2" setuptools_scm build twine
+
+      shell: bash
+
+    - name: Build binary wheel
+      run: python -m build --sdist --wheel . --outdir dist
```

Comment:
Wheel will fail b/c we don't have wheels for `cf_units`. Udunits2 make it hard to create one. I'm working on a solution upstream.

Reply:
I just recently got `cf_units` built under pip and it was pretty nightmarish to build.  I'll probably update the build instructions very soon.

Reply:
We are getting somewhere in https://github.com/SciTools/cf-units/pull/186 and the wheel should make this easier.